monkey/agent_plugins/exploiters/mssql/src/mssql_options.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from pydantic import Field
+
+from common.base_models import InfectionMonkeyBaseModel
+from common.types import NetworkPort
+
+
+class MSSQLOptions(InfectionMonkeyBaseModel):
+    target_ports: List[NetworkPort] = Field(
+        default=[1433],
+        description="A list of TCP ports that the MSSQL exploiter will try to exploit.",
+    )
+    try_all_discovered_tcp_ports: bool = Field(
+        default=False,
+        description="Attempt to exploit MSSQL on all TCP ports discovered from network scanning.",
+    )
+    query_timeout: float = Field(
+        gt=0.0,
+        default=15.0,
+        description="The maximum time (in seconds) to wait for response from the MSSQL server.",
+    )
+    agent_binary_download_timeout: float = Field(
+        gt=0.0,
+        default=60.0,
+        description="The maximum time (in seconds) to wait for a successfully exploited"
+        " MSSQL server to download the agent binary.",
+    )
+    login_timeout: float = Field(
+        gt=0.0,
+        default=15.0,
+        description="The maximum time (in seconds) to wait for the SQL Server login to complete",
+    )
```

Comment:
Is there any known reason to distinguish this from `query_timeout`? Perhaps we could have just one "timeout" that is the amount of time to wait for a response from the server. Whether it's a login response or a query response probably doesn't matter.

Reply:
I thought that make sense. In either case both of them raise the same error if we pass the timeout so I don't think it really matters. I will merge them and call its `server_timeout`